gradlew

```diff
@@ -1,7 +1,7 @@
-#!/bin/sh
```

Comment:
What's with the empty wrapper jar?!

Reply:
I don't know why it says it's empty - the file is about 60k on my branch (https://github.com/frenziedherring/logcat/blob/frenziedherring/logcatx/gradle/wrapper/gradle-wrapper.jar)